File: app/src/main/java/com/babestudios/companyinfouk/data/model/charges/ChargesItem.java
package com.babestudios.companyinfouk.data.model.charges;


import com.google.gson.annotations.SerializedName;

import java.util.ArrayList;
import java.util.List;

public class ChargesItem {
	@SerializedName("acquired_on")
	public String acquiredOn;
	@SerializedName("assets_ceased_released")
	public String assetsCeasedReleased;
	@SerializedName("charge_code")
	public String chargeCode;
	@SerializedName("charge_number")
	public String chargeNumber;
	@SerializedName("classification")
	public Classification classification;
	@SerializedName("covering_instrument_date")
	public String coveringInstrumentDate;
	@SerializedName("created_on")
	public String createdOn;
	@SerializedName("delivered_on")
	public String deliveredOn;
	@SerializedName("etag")
	public String etag;
	@SerializedName("id")
	public String id;
	@SerializedName("insolvency_cases")
	public List<InsolvencyCase> insolvencyCases = new ArrayList<>();
	@SerializedName("links")
	public ChargesItemLinks links;
	@SerializedName("more_than_four_persons_entitled")
	public String moreThanFourPersonsEntitled;
	@SerializedName("particulars")
	public Particulars particulars;
	@SerializedName("persons_entitled")
	public List<PersonsEntitled> personsEntitled = new ArrayList<>();
	@SerializedName("resolved_on")
	public String resolvedOn;
	@SerializedName("satisfied_on")
	public String satisfiedOn;
	@SerializedName("scottish_alterations")
	public ScottishAlterations scottishAlterations;
	@SerializedName("secured_details")
	public SecuredDetails securedDetails;
	@SerializedName("status")
	public String status;
	@SerializedName("transactions")
	public List<Transaction> transactions = new ArrayList<>();
}

File: app/src/main/java/com/babestudios/companyinfouk/data/model/charges/ChargesItem.kt
package com.babestudios.companyinfouk.data.model.charges


import com.google.gson.annotations.SerializedName

import java.util.ArrayList

class ChargesItem {
	@SerializedName("acquired_on")
	var acquiredOn: String? = null
	@SerializedName("assets_ceased_released")
	var assetsCeasedReleased: String? = null
	@SerializedName("charge_code")
	var chargeCode: String? = null
	@SerializedName("charge_number")
	var chargeNumber: String? = null
	@SerializedName("classification")
	var classification: Classification? = null
	@SerializedName("covering_instrument_date")
	var coveringInstrumentDate: String? = null
	@SerializedName("created_on")
	var createdOn: String? = null
	@SerializedName("delivered_on")
	var deliveredOn: String? = null
	@SerializedName("etag")
	var etag: String? = null
	@SerializedName("id")
	var id: String? = null
	@SerializedName("insolvency_cases")
	var insolvencyCases: List<InsolvencyCase> = ArrayList()
	@SerializedName("links")
	var links: ChargesItemLinks? = null
	@SerializedName("more_than_four_persons_entitled")
	var moreThanFourPersonsEntitled: String? = null
	@SerializedName("particulars")
	var particulars: Particulars? = null
	@SerializedName("persons_entitled")
	var personsEntitled: List<PersonsEntitled> = ArrayList()
	@SerializedName("resolved_on")
	var resolvedOn: String? = null
	@SerializedName("satisfied_on")
	var satisfiedOn: String? = null
	@SerializedName("scottish_alterations")
	var scottishAlterations: ScottishAlterations? = null
	@SerializedName("secured_details")
	var securedDetails: SecuredDetails? = null
	@SerializedName("status")
	var status: String? = null
	@SerializedName("transactions")
	var transactions: List<Transaction> = ArrayList()
}
